real_time/surface_src/java_src/BuggyRos/src/test/java/UnboundedMessageQueueTest.java

```diff
@@ -0,0 +1,76 @@
+import static org.junit.Assert.fail;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class UnboundedMessageQueueTest {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2362871863/issues/source?bid=2938665&fileBranchId=3220358#l7)